src/server/middleware/setup.ts

```diff
@@ -0,0 +1,16 @@
+/* First setup of wg-easy app */
+export default defineEventHandler(async (event) => {
+  const url = getRequestURL(event);
+
+  if (
+    url.pathname.startsWith('/setup') ||
+    url.pathname === '/api/account/new'
+  ) {
+    return;
+  }
+
+  const users = await Database.getUsers();
+  if (users.length === 0) {
+    return sendRedirect(event, '/setup', 302);
```

Comment:
this runs on every request but is only needed once. is there a better way to do this?